### ISSUE TITLE
Unable to click the edit button of alarm 

### ISSUE THREAD

**Issue by Zesko** [zagortenay333/timepp#197]
The current latest version (fixed closing fullscreen of notification)  does not let me to click the edit button of alarm. The "edit" button disappears when the cursor of mouse touches it.

See the screenshot
When the cursor touches any entry then the edit button appears:
![image](https://user-images.githubusercontent.com/17826352/178813116-4b92abd3-6ad6-4a47-870a-1f7bac0a09f1.png)

But the cursor touches this edit button that will disappear:
![image](https://user-images.githubusercontent.com/17826352/178813277-22c96416-7b48-42fb-a526-3cf9d89ea910.png)

Gnome: 42.3.1 Wayland
GTK: 3.24.34
GDM: 42.0
Dash to Panel version 50

**Comment by Zesko** [comment]
There is another issue:

Time setting does not display when the cursor moves from this extension to other extension in the same panel, then moving back to this extension. 
![image](https://user-images.githubusercontent.com/17826352/178814335-2e4f8cef-929d-4e42-95c0-cb93c044d6a3.png)

**Comment by c-git** [comment]
I was able to reproduce the menu not showing after moving the mouse away and then back. For me if I move to one of the others (like pomodoro or timer they wouldn't show unless I click them). I further noted that the same would happen to any of the other components like timer.)

See video showing the problem <https://cdn.discordapp.com/attachments/850255840875118672/996995523535970394/recording.mp4>


![image](https://user-images.githubusercontent.com/43485962/178896078-ae64a16a-06b8-4d6b-b89e-58df07bcdff8.png)

**Comment by zagortenay333** [comment]
Unfortunately I'm not on gnome shell and my crappy laptop doesn't support virtualization so I can't debug this.

Run this in a terminal:

```
journalctl /usr/bin/gnome-shell -r
```

..then replicate the bug and see what gets reported.

**Comment by Zesko** [comment]
`journalctl /usr/bin/gnome-shell --no-page -b -f`


```
Jul 14 10:54:01 p15v gnome-shell[1678]: clutter_actor_set_mapped: assertion '!CLUTTER_ACTOR_IN_MAP_UNMAP (self)' failed
Jul 14 10:54:03 p15v gnome-shell[1678]: clutter_actor_set_mapped: assertion '!CLUTTER_ACTOR_IN_MAP_UNMAP (self)' failed
Jul 14 10:54:11 p15v gnome-shell[1678]: clutter_actor_set_mapped: assertion '!CLUTTER_ACTOR_IN_MAP_UNMAP (self)' failed
Jul 14 10:54:12 p15v gnome-shell[1678]: clutter_actor_set_mapped: assertion '!CLUTTER_ACTOR_IN_MAP_UNMAP (self)' failed
```

This is exactly message that appears when the cursor touches the "edit" button (Tried 4 times = 4 same messages).

**Comment by digitalillusion** [comment]
seems an issue of gnome-shell https://bugzilla.redhat.com/show_bug.cgi?id=2069975

**Comment by digitalillusion** [comment]
Seems an issue of gnome-shell: 

https://gitlab.gnome.org/GNOME/mutter/-/merge_requests/2299

the fix will be available in gnome 42.1 according to what said in the related issue

https://gitlab.gnome.org/GNOME/gnome-shell/-/issues/3165

**Comment by zagortenay333** [comment]
Fixed.
